domains/runtime/evm/src/lib.rs

```diff
@@ -1376,6 +1437,21 @@ impl_runtime_apis! {
                 .saturating_add(Weight::from_parts(0, len))
         }
 
+        fn extrinsics_weight(extrinsics: &Vec<<Block as BlockT>::Extrinsic>) -> Weight {
```

Comment:
some of these functions can be common across runtimes. Ideal if we can move them to runtime primitives of other common crate

Reply:
The code is mostly the same, in fact almost every runtime API implementation looks like the same cross domain runtime, but they involve runtime-specific variables e.g. `<Block as BlockT>::Extrinsic>` here referring to concrete extrinsic type that contains `RuntimeCall` generated by `construct_runtime!` while in other places like `sp-domains` it refers to `OpaqueExtrinsic`.

I not look close to every runtime API, may be there are code that can be extracted and reused, but not really the purpose of this PR.